src/mlpack/methods/ne/gene.hpp

```diff
@@ -0,0 +1,186 @@
+/**
+ * @file gene.hpp
+ * @author Bang Liu
+ *
+ * Definition of the LinkGene and NeuronGene class.
+ */
```

Comment:
@zoq Yeah, I can fix this. I put them into one file because they are all genes, and highly related to each other.